.github/workflows/maven.yml

```
@@ -300,6 +306,21 @@ jobs:
           # Counter-intuitive BUT trivy-action has its own cache which duplicates our own but in a less flexible way
           cache: false
 
+      - name: Upload JaCoCo Report
+        if: ${{ matrix.os == 'ubuntu-latest'  && inputs.PUBLISH_JACOCO_REPORT }}
+        uses: actions/upload-artifact@v4.4.3
+        with:
+          name: jacoco-report
+          path: target/site/jacoco/
```

```suggestion
          path: |
            target/site/jacoco/
            **/target/site/jacoco/
```
Think this wants the a second pattern with the leading `**/` glob to find JaCoCo reports in all sub-modules, otherwise we'd only pick up the top level module reports which in a multi-module build is probably the least interesting one/may not exist at all as the top level module is often just the parent